Python/VisualizeTS_Agility.py

```diff
@@ -363,6 +362,7 @@ def calcFullTrialStarts(fileName, filePath):
 plt.tight_layout()
 
 ## TODO: extend plots to include:
-### sagittal moments & COM power by making plot a function. 
+### Make a new plot for Sagittal moments & COM power 
```

Comment:
COM Power wasn't exported for these trials

Reply:
Make sagittal moments for now. Or bonus challenge: calculate COM power from force data